index.html

```diff
@@ -84,5 +84,162 @@ <h1>
         </ul>
       </section>
     </main>
+    <section class="popup-window invisible"></section>
+    <section id="Portfolio" class="recent-work">
+      <header>
+        <h1 class="section-header">My Recent Works</h1>
+        <div class="indicator"></div>
+      </header>
+      <div class="grid-container">
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+        <div class="item">
+          <div class="placeholder"></div>
+          <div class="text">
+            <h3>To-do List</h3>
+            <ul class="skillsbar">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button class="popup">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="About">
+      <div class="aboutme-text">
+        <div class="head-ele">
+          <h1 class="aboutme-header">About Myself</h1>
```

Comment:
- [ ] Remember that there should only be one `<h1>` tag per page and it is already in use in the headline section. You could use the `<h2>` for each section title :wink: